api/models/Advertisement.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema
+
+const AdvertisementSchema = new Schema(
+  {
+    pictureURL: {
```

Comment:
To keep variables camelcase, lets do 
```suggestion
    pictureUrl: {
```

Reply:
youll have to change this everywhere else too